### PR TITLE
[ocean] Fix mappings to index Meetup data

### PR DESCRIPTION
**Pull request by valeriocos**
This patch prevents to index the content of comments and author bios, which may cause failures when uploading the data to ES

### CODE REVIEW

File: grimoire_elk/elk/meetup.py [hunk 1]
```diff
@@ -43,38 +43,25 @@ def get_elastic_mappings(es_major):
         :param es_major: major version of Elasticsearch, as string
         :returns:        dictionary with a key, 'items', with the mapping
         """
-
-        if es_major != '2':
-            mapping = """
```
Comment (acs) [inline grimoire_elk/elk/meetup.py]: I won't remove the ES 2.x support because it can cause regressions.

**Review by acs — CHANGES_REQUESTED** [review]

**Review by acs — COMMENTED** [review]
Hmmm, I can not see ES 2.x support readded.

**Comment by valeriocos** [comment]
I have just updated the PR

**Review by acs — APPROVED** [review]
LGTM

**Pull request merged**